src/conf_mode/vpn_ipsec.py

```diff
@@ -104,6 +108,8 @@ def get_config(config=None):
     ipsec = config_dict_merge(default_values, ipsec)
 
     ipsec['dhcp_interfaces'] = set()
+    ipsec['enabled_vti_interfaces'] = set()
```

Comment:
Why do we need an empty set here? Usually, we go like: Feature is only set if key is found in dictionary.

Reply:
This is the same as `ipsec['dhcp_interfaces']` directly above, I am using a set to accumulate a list of interfaces (if any) that have the feature enabled as I scan through the config. I could not initialize this here, but I would then have to lazy-initialize it at each place I might want to add something to the set.

Reply:
Yes, you followed an existing patter which is good and bad at the same time. Lets keep it this way for now, but we should get rid if this in the future and really rely on key presence in the dict.